% Template for PLoS
% Version 3.5 March 2018
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that 
% no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use the PLoS provided BiBTeX style
\bibliographystyle{plos2015}
% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother



% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
%\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
%\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\today}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Beta series correlations} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
James Kent\textsuperscript{1*},
Michelle Voss\textsuperscript{1},
%Name3 Surname\textsuperscript{2,3\textcurrency},
%Name4 Surname\textsuperscript{2},
%Name5 Surname\textsuperscript{2\ddag},
%Name6 Surname\textsuperscript{2\ddag},
%Name7 Surname\textsuperscript{1,2,3*},
%with the Lorem Ipsum Consortium\textsuperscript{\textpilcrow}
\\
\bigskip
\textbf{1} Psychology Department, University of Iowa, Iowa City, Iowa, United States
%\\
%\textbf{2} Affiliation Dept/Program/Center, Institution Name, City, State, Country
%\\
%\textbf{3} Affiliation Dept/Program/Center, Institution Name, City, State, Country
%\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
% \textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
% \dag Deceased

% Group/Consortium Author Note
% \textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* james-kent@uiowa.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
Simulations, task, and combination

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLOS ONE authors please skip this step. 
% Author Summary not valid for PLOS ONE submissions.   
% \section*{Author summary}
% Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur eget porta erat. Morbi consectetur est vel gravida pretium. Suspendisse ut dui eu ante cursus gravida non sed sem. Nullam sapien tellus, commodo id velit id, eleifend volutpat quam. Phasellus mauris velit, dapibus finibus elementum vel, pulvinar non tellus. Nunc pellentesque pretium diam, quis maximus dolor faucibus id. Nunc convallis sodales ante, ut ullamcorper est egestas vitae. Nam sit amet enim ultrices, ultrices elit pulvinar, volutpat risus.

\linenumbers

% Use "Eq" instead of "Equation" for equation citations.
\section*{Introduction}
\subsection{Concepts To Cover}
\begin{enumerate}
	\item Big Idea
  \begin{enumerate}
    \item Evaluate beta series correlations between brain regions during a task.
  \end{enumerate}
	\item Topics
	\begin{enumerate}
    \item Definition of a trial
    \item General Linear Model
    \item Block versus rapid event related designs
    \item BOLD variability
    \item BOLD Response/BOLD correlations
    \item Beta Series Correlations and PsychoPhysiological Interactions
    \item Different methods to estimate beta series correlations: Least-Squares All and Least-Squares Separate
  \end{enumerate}
	\item Gap
  \begin{enumerate}
    \item combination of simulation/real data to compare/validate methods
  \end{enumerate}
\end{enumerate}

\subsection{Introduction}
What are the available methods to measure which brain regions correlate during a task with functional Magnetic Resonance Imaging (fMRI)?
The two most popular methods appear to be Psychophysiological Interactions (PPI) and Beta Series Correlations (BSC).
However, the efficacy of PPI and BSC depends on the experimental design.
Many experimental designs place trials belonging to different conditions close to each other
to capture the psychological process of interest.
For example, if a task measured surprise, placing all the surprising stimuli together
invalidates the concept one is attempting to measure.
These rapid event-related experimental designs with intermixed conditions (?) are difficult to analyze.
The difficulty comes from the sluggish Blood Oxygen Level Dependent (BOLD) response.
Multiple trials overlap as they occur faster than the time it takes the BOLD response to return to baseline.
This overlap creates ambiguity whether the changing BOLD response is due to the trial of interest or surrounding trials.
Previous work has found PPI is less powerful than BSC to detect condition differences using rapid event-related designs.
Thus, only variants of BS will be considered.
While previous studies have investigated variants of BS, they either only focused on multivariate pattern analysis
or only performed simulations.
We will focus on BSC using both simulations and real data to compare and validate methods,
expanding and improving previous work.
Specifically, we will compare two strategies to generate BS's: "Least-Squares All" (LSA) and "Least-Squares Separate" (LSS).
LSA has a regressor for each individual trial in the General Linear Model (GLM).
% Lorem ipsum dolor sit~\cite{bib1} amet, consectetur adipiscing elit. Curabitur eget porta erat. Morbi consectetur est vel gravida pretium. Suspendisse ut dui eu ante cursus gravida non sed sem. Nullam Eq~(\ref{eq:schemeP}) sapien tellus, commodo id velit id, eleifend volutpat quam. Phasellus mauris velit, dapibus finibus elementum vel, pulvinar non tellus. Nunc pellentesque pretium diam, quis maximus dolor faucibus id.~\cite{bib2} Nunc convallis sodales ante, ut ullamcorper est egestas vitae. Nam sit amet enim ultrices, ultrices elit pulvinar, volutpat risus.
\subsection{Understanding brain organization}
\begin{enumerate}
  \item evolving relationship between resting state and task fmri
  \item connectivity in task fmri is measured in different ways
  \item linear addition of task activation on top of low frequency oscillations
  \item other view that oscillations change in complex ways
  \item could be variances in bold response independent of the low frequency oscillations
\end{enumerate}

\subsection{How to measure brain organization}
- methods include: psychophysiological interactions and beta series correlations

\subsection{comparison of LSS and LSA methods}
% \begin{eqnarray}
% \label{eq:schemeP}
% 	\mathrm{P_Y} = \underbrace{H(Y_n) - H(Y_n|\mathbf{V}^{Y}_{n})}_{S_Y} + \underbrace{H(Y_n|\mathbf{V}^{Y}_{n})- H(Y_n|\mathbf{V}^{X,Y}_{n})}_{T_{X\rightarrow Y}},
% \end{eqnarray}

\section*{Materials and methods}
\subsection*{Beta Series Correlation Simulations}
- Time Repetition = 2
- duration of experiment = variable
- intertrial interval = 2 - 8 seconds (exponential sampling)
- response scaled peak response is 1
- glover HRF
% https://github.com/brainiak/brainiak/blob/dc12929a99529a69da3c3242e9c9fd31c308fbb0/brainiak/utils/fmrisim.py#L722
- the height of the voxel HRF responses was drawn from a multivariate normal distribution with mean 1 and standard deviation of 1.
% https://github.com/jdkent/betaSeriesSimulations/blob/2fc45bcbc6873d395432d01359ba492b6f3b1458/beta_sim/interfaces/fmrisim.py#L204
% https://github.com/jdkent/betaSeriesSimulations/blob/2fc45bcbc6873d395432d01359ba492b6f3b1458/beta_sim/interfaces/fmrisim.py#L170
% - noise was added through:
%     - snr: 40
%     - sfnr: 60
%     - task_sigma $1: Size of the variance of task specific noise
%     - drift_sigma 1: Size of the variance of drift noise
%     - auto_reg_sigma 0.5: Size of the variance of autoregressive
%         noise. This is an ARMA process where the AR and MA components can be
%         separately specified.
%     - physiological_sigma 1: Size of the variance of physiological
%         noise
%     - auto_reg_rho [0.5]: The coefficients of the autoregressive
%         components you are modeling
%     - ma_rho [0.0]:The coefficients of the moving average components you
%         are modeling
- voxels correlated at set correlation values (0.1, 0.2, 0.3)
- betas fit by least squares
- N=10000 (at a given noise level)
- start and end of sessions were not ignored (optimal experimental design)
- highpass filter: 1/128 hz


- varying inter-trial-interval
- varying noise level
- varying number of trials

\subsection*{Task Switching Validation}
- 61 participants (31 females)
- 65+ age: 71.754098 mean, 4.766745 std
- education: 17.065574 mean, 2.657498 std
- task: task switching, 5 blocks
- single task (odd/even) (high/low) (30 trials each block)
- 3 mixed blocks (48 repeat/ 40 switch trials)
- 6 30 second rest blocks
- participants had 4 practice trials immediately before beginning
- participants practiced the task out of scanner.
- iti average 3.5 seconds with exponential distribution
- TR: 2 seconds, te: 30 ms

\subsection{Preparing fMRI}
Results included in this manuscript come from preprocessing performed
using \emph{fMRIPrep} 1.5.7 (\cite{fmriprep1}; \cite{fmriprep2}; RRID:SCR\_016216),
which is based on \emph{Nipype} 1.4.0
(\cite{nipype1}; \cite{nipype2}; RRID:SCR\_002502).

\begin{description}
\item[Anatomical data preprocessing]
The T1-weighted (T1w) image was corrected for intensity non-uniformity
(INU) with \texttt{N4BiasFieldCorrection} \cite{n4}, distributed with
ANTs 2.2.0 \cite[RRID:SCR\_004757]{ants}, and used as T1w-reference
throughout the workflow. The T1w-reference was then skull-stripped with
a \emph{Nipype} implementation of the \texttt{antsBrainExtraction.sh}
workflow (from ANTs), using OASIS30ANTs as target template. Brain tissue
segmentation of cerebrospinal fluid (CSF), white-matter (WM) and
gray-matter (GM) was performed on the brain-extracted T1w using
\texttt{fast} \cite{fsl_fast} [FSL 5.0.9, RRID:SCR\_002823,][]. Brain
surfaces were reconstructed using \texttt{recon-all} \cite{fs_reconall},
[FreeSurfer 6.0.1, RRID:SCR\_001847,][] and the brain mask estimated
previously was refined with a custom variation of the method to
reconcile ANTs-derived and FreeSurfer-derived segmentations of the
cortical gray-matter of Mindboggle
\cite[RRID:SCR\_002438,]{mindboggle}. Volume-based spatial
normalization to two standard spaces (MNI152NLin2009cAsym,
MNI152NLin6Asym) was performed through nonlinear registration with
\texttt{antsRegistration} (ANTs 2.2.0), using brain-extracted versions
of both T1w reference and the T1w template. The following templates were
selected for spatial normalization: \emph{ICBM 152 Nonlinear
Asymmetrical template version 2009c} {[}\cite{mni152nlin2009casym},
RRID:SCR\_008796; TemplateFlow ID: MNI152NLin2009cAsym{]}, \emph{FSL's
MNI ICBM 152 non-linear 6th Generation Asymmetric Average Brain
Stereotaxic Registration Model} {[}\cite{mni152nlin6asym},
RRID:SCR\_002823; TemplateFlow ID: MNI152NLin6Asym{]}.
\item[Functional data preprocessing]
For each of the 4 BOLD runs found per subject (across all tasks and
sessions), the following preprocessing was performed. First, a reference
volume and its skull-stripped version were generated using a custom
methodology of \emph{fMRIPrep}. Susceptibility distortion correction
(SDC) was omitted. The BOLD reference was then co-registered to the T1w
reference using \texttt{bbregister} (FreeSurfer) which implements
boundary-based registration \cite{bbr}. Co-registration was configured
with six degrees of freedom. Head-motion parameters with respect to the
BOLD reference (transformation matrices, and six corresponding rotation
and translation parameters) are estimated before any spatiotemporal
filtering using \texttt{mcflirt} \cite[FSL 5.0.9,]{mcflirt}. BOLD
runs were slice-time corrected using \texttt{3dTshift} from AFNI
20160207 \cite[RRID:SCR\_005927]{afni}. The BOLD time-series, were
resampled to surfaces on the following spaces: \emph{fsaverage5}. The
BOLD time-series (including slice-timing correction when applied) were
resampled onto their original, native space by applying the transforms
to correct for head-motion. These resampled BOLD time-series will be
referred to as \emph{preprocessed BOLD in original space}, or just
\emph{preprocessed BOLD}. The BOLD time-series were resampled into
several standard spaces, correspondingly generating the following
\emph{spatially-normalized, preprocessed BOLD runs}:
MNI152NLin2009cAsym, MNI152NLin6Asym. First, a reference volume and its
skull-stripped version were generated using a custom methodology of
\emph{fMRIPrep}. Automatic removal of motion artifacts using independent
component analysis \cite[ICA-AROMA,]{aroma} was performed on the
\emph{preprocessed BOLD on MNI space} time-series after removal of
non-steady state volumes and spatial smoothing with an isotropic,
Gaussian kernel of 6mm FWHM (full-width half-maximum). Corresponding
``non-aggresively'' denoised runs were produced after such smoothing.
Additionally, the ``aggressive'' noise-regressors were collected and
placed in the corresponding confounds file. Several confounding
time-series were calculated based on the \emph{preprocessed BOLD}:
framewise displacement (FD), DVARS and three region-wise global signals.
FD and DVARS are calculated for each functional run, both using their
implementations in \emph{Nipype} \cite[following the definitions
by]{power_fd_dvars}. The three global signals are extracted within the
CSF, the WM, and the whole-brain masks. Additionally, a set of
physiological regressors were extracted to allow for component-based
noise correction \cite[\emph{CompCor},]{compcor}. Principal
components are estimated after high-pass filtering the
\emph{preprocessed BOLD} time-series (using a discrete cosine filter
with 128s cut-off) for the two \emph{CompCor} variants: temporal
(tCompCor) and anatomical (aCompCor). tCompCor components are then
calculated from the top 5\% variable voxels within a mask covering the
subcortical regions. This subcortical mask is obtained by heavily
eroding the brain mask, which ensures it does not include cortical GM
regions. For aCompCor, components are calculated within the intersection
of the aforementioned mask and the union of CSF and WM masks calculated
in T1w space, after their projection to the native space of each
functional run (using the inverse BOLD-to-T1w transformation).
Components are also calculated separately within the WM and CSF masks.
For each CompCor decomposition, the \emph{k} components with the largest
singular values are retained, such that the retained components' time
series are sufficient to explain 50 percent of variance across the
nuisance mask (CSF, WM, combined, or temporal). The remaining components
are dropped from consideration. The head-motion estimates calculated in
the correction step were also placed within the corresponding confounds
file. The confound time series derived from head motion estimates and
global signals were expanded with the inclusion of temporal derivatives
and quadratic terms for each \cite{confounds_satterthwaite_2013}.
Frames that exceeded a threshold of 0.5 mm FD or 1.5 standardised DVARS
were annotated as motion outliers. All resamplings can be performed with
\emph{a single interpolation step} by composing all the pertinent
transformations (i.e.~head-motion transform matrices, susceptibility
distortion correction when available, and co-registrations to anatomical
and output spaces). Gridded (volumetric) resamplings were performed
using \texttt{antsApplyTransforms} (ANTs), configured with Lanczos
interpolation to minimize the smoothing effects of other kernels
\cite{lanczos}. Non-gridded (surface) resamplings were performed using
\texttt{mri\_vol2surf} (FreeSurfer).
\end{description}

Many internal operations of \emph{fMRIPrep} use \emph{Nilearn} 0.6.1
\cite[RRID:SCR\_001362]{nilearn}, mostly within the functional
processing workflow. For more details of the pipeline, see
\href{https://fmriprep.readthedocs.io/en/latest/workflows.html}{the
section corresponding to workflows in \emph{fMRIPrep}'s documentation}.

\subsection{BetaSeries Correlations}

Results included in this manuscript come from modeling performed using
\emph{NiBetaSeries} 0.4.3 \cite{Kent2018}, which is based on
\emph{Nipype} 1.4.2 \cite{Gorgolewski2011, Gorgolewski2018}.

\hypertarget{beta-series-modeling}{%
\subsubsection{Beta Series Modeling}\label{beta-series-modeling}}

Least squares- separate (LSS) models were generated for each event in
the task following the method described in \cite{Turner2012a}, using
Nistats 0.0.1b1.\\
Prior to modeling, preprocessed data were masked,and mean-scaled over
time. For each trial, preprocessed data were subjected to a general
linear model in which the trial was modeled with its own regressor, while
all other trials from that condition were modeled in a second regressor,
and other conditions were modeled in their own regressors. Each
condition regressor was convolved with a \cite[glover hemodynamic response
function]{Glover1999}.\\
In addition to condition regressors, average white matter signal, average csf signal,
cosine basis set high pass regressors, the initial four non stead state volumes,
\cite[friston 24]{Friston1996}, and motion outliers were included
in the model. AR(1) prewhitening was applied in each model to account
for temporal autocorrelation.

After fitting each model, the parameter estimate (i.e., beta) map
associated with the target trial's regressor was retained and
concatenated into a 4D image with all other trials from the same
condition, resulting in a set of N 4D images where N refers to the
number of conditions in the task. The number of volumes in each 4D image
represents the number of trials in that condition.

\hypertarget{atlas-connectivity-analysis}{%
\subsubsection{Atlas Connectivity
Analysis}\label{atlas-connectivity-analysis}}

The beta series 4D image for each condition in the task was subjected to
an ROI-to-ROI connectivity analysis to produce a condition-specific
correlation matrix.
Two atlases were used to generate ROI-to-ROI connectivity matrices.
The \cite[Schaefer Atlas (400 parcels, 17 networks)]{Schaefer2017} was
used to comprehensively cover the cortex, and a custom atlas representing
statistically significant clusters of voxels that responded across all trial types
in task switching.

The correlation coefficient estimator used for this
step was empirical covariance, as implemented in Nilearn 0.4.2
\cite{Abraham2014}. Correlation coefficients were converted to
normally-distributed z-values using Fisher's r-to-z conversion
\cite{Fisher1915}. Figures for the correlation matrices were generated
with Matplotlib 2.2.5 \cite{Hunter2007} and MNE-Python 0.18.2
\cite{Gramfort2013, Gramfort2014}.

\hypertarget{software-dependencies}{%
\subsubsection{Software Dependencies}\label{software-dependencies}}

Additional libraries used in the NiBetaSeries workflow include
\emph{Pybids} 0.9.5 \cite{Yarkoni2019}, \emph{Niworkflows} 1.0.4,
\emph{Nibabel} 2.4.1, \emph{Pandas} 0.24.2 \cite{McKinney2010}, and
\emph{Numpy} 1.18.1 \cite{VanDerWalt2011, Oliphant2006}.

\subsection*{Traditional Task Switch Analysis/Atlas Generation}
\begin{enumerate}
  \item use similar confounds (replaced friston 24 with framewise displacement
        (may wish to use framewise displacement with betaseries as well?)
  \item used smoothing kernel of 6mm
  \item generated contrasts for each trial type relative to baseline,
        an overall contrast, and "switch-repeat" (should include "repeat-single" as well?)
  \item second level contrast from all the first level contrasts.
\end{enumerate}
\subsection*{contrast noise ratio calculation}
\begin{enumerate}
  \item use finite impulse responses (FIR), observing the amplitude 6 second post onset
  \item take 90th percentile of significant responses
  \item use the extracted cnr from the data to inform cnr for simulations (25th percentile, 50th percentile, 75 percentile)
\end{enumerate}
\subsection*{Task Switching Beta Series Correlation Simulations}
- use real task (Task Switch) to validate the simulations
- generate contrast noise ratios for each image in the real dataset
    - vary contrast noise ratios at 25th, 50th, and 75th percentiles
- simulate beta series
- issues:
    - the cnr does not exactly match the comparisons I will be performing
        - e.g., the cnr is based on an average of all trial types,
          while the contrasts on real data will be done for specific trial types



\subsection*{Task Switching Beta Series Correlation Analysis}

% For figure citations, please use "Fig" instead of "Figure".
% Nulla mi mi, Fig~\ref{fig1} venenatis sed ipsum varius, volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, \nameref{S1_Video} vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero.

% Place figure captions after the first paragraph in which they are cited.
% \begin{figure}[!h]
% \caption{{\bf Bold the figure title.}
% Figure caption text here, please use this space for the figure panel descriptions instead of using subfigure commands. A: Lorem ipsum dolor sit amet. B: Consectetur adipiscing elit.}
% \label{fig1}
% \end{figure}

% Results and Discussion can be combined.
\section*{Results}

\subsection*{Beta Series Correlation Simulations}

\begin{enumerate}
  \item power analysis (is the null effect maintained?)/ can we detect a 0.1 difference?
  \item is the signal biased? (e.g. systematic over or under estimation)
\end{enumerate}

\subsection*{Task Switching Validation}
% Nulla mi mi, venenatis sed ipsum varius, Table~\ref{table1} volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero.
- traditional substraction does not reveal differences (with full atlas)
- will see whether subtraction with task activation atlas has the same result

\subsection*{Task Switching Beta Series Correlation Simulations}

\subsection*{Task Switching Beta Series Correlations}
% Place tables after the first paragraph in which they are cited.
% \begin{table}[!ht]
% \begin{adjustwidth}{-2.25in}{0in} % Comment out/remove adjustwidth environment if table fits in text column.
% \centering
% \caption{
% {\bf Table caption Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam.}}
% \begin{tabular}{|l+l|l|l|l|l|l|l|}
% \hline
% \multicolumn{4}{|l|}{\bf Heading1} & \multicolumn{4}{|l|}{\bf Heading2}\\ \thickhline
% $cell1 row1$ & cell2 row 1 & cell3 row 1 & cell4 row 1 & cell5 row 1 & cell6 row 1 & cell7 row 1 & cell8 row 1\\ \hline
% $cell1 row2$ & cell2 row 2 & cell3 row 2 & cell4 row 2 & cell5 row 2 & cell6 row 2 & cell7 row 2 & cell8 row 2\\ \hline
% $cell1 row3$ & cell2 row 3 & cell3 row 3 & cell4 row 3 & cell5 row 3 & cell6 row 3 & cell7 row 3 & cell8 row 3\\ \hline
% \end{tabular}
% \begin{flushleft} Table notes Phasellus venenatis, tortor nec vestibulum mattis, massa tortor interdum felis, nec pellentesque metus tortor nec nisl. Ut ornare mauris tellus, vel dapibus arcu suscipit sed.
% \end{flushleft}
% \label{table1}
% \end{adjustwidth}
% \end{table}


%PLOS does not support heading levels beyond the 3rd (no 4th level headings).
% \subsection*{\lorem\ and \ipsum\ nunc blandit a tortor}
% \subsubsection*{3rd level heading} 
% Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque. Quisque augue sem, tincidunt sit amet feugiat eget, ullamcorper sed velit. Sed non aliquet felis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris commodo justo ac dui pretium imperdiet. Sed suscipit iaculis mi at feugiat. 

% \begin{enumerate}
% 	\item{react}
% 	\item{diffuse free particles}
% 	\item{increment time by dt and go to 1}
% \end{enumerate}


% \subsection*{Sed ac quam id nisi malesuada congue}

% Nulla mi mi, venenatis sed ipsum varius, volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero.

% \begin{itemize}
% 	\item First bulleted item.
% 	\item Second bulleted item.
% 	\item Third bulleted item.
% \end{itemize}

\section*{Discussion}
% Nulla mi mi, venenatis sed ipsum varius, Table~\ref{table1} volutpat euismod diam. Proin rutrum vel massa non gravida. Quisque tempor sem et dignissim rutrum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at justo vitae nulla elementum commodo eu id massa. In vitae diam ac augue semper tincidunt eu ut eros. Fusce fringilla erat porttitor lectus cursus, vel sagittis arcu lobortis. Aliquam in enim semper, aliquam massa id, cursus neque. Praesent faucibus semper libero~\cite{bib3}.
\subsection*{Simulation Conclusions}

\subsection*{Task Switching Conclusions}

\subsection{Limitations}
\section*{Conclusion}

% CO\textsubscript{2} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque. Quisque augue sem, tincidunt sit amet feugiat eget, ullamcorper sed velit. 

% Sed non aliquet felis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris commodo justo ac dui pretium imperdiet. Sed suscipit iaculis mi at feugiat. Ut neque ipsum, luctus id lacus ut, laoreet scelerisque urna. Phasellus venenatis, tortor nec vestibulum mattis, massa tortor interdum felis, nec pellentesque metus tortor nec nisl. Ut ornare mauris tellus, vel dapibus arcu suscipit sed. Nam condimentum sem eget mollis euismod. Nullam dui urna, gravida venenatis dui et, tincidunt sodales ex. Nunc est dui, sodales sed mauris nec, auctor sagittis leo. Aliquam tincidunt, ex in facilisis elementum, libero lectus luctus est, non vulputate nisl augue at dolor. For more information, see \nameref{S1_Appendix}.

\section*{Supporting information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
% \paragraph*{S1 Fig.}
% \label{S1_Fig}
% {\bf Bold the title sentence.} Add descriptive text after the title of the item (optional).

% \paragraph*{S2 Fig.}
% \label{S2_Fig}
% {\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 File.}
% \label{S1_File}
% {\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 Video.}
% \label{S1_Video}
% {\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 Appendix.}
% \label{S1_Appendix}
% {\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 Table.}
% \label{S1_Table}
% {\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\section*{Acknowledgments}
%Cras egestas velit mauris, eu mollis turpis pellentesque sit amet. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam id pretium nisi. Sed ac quam id nisi malesuada congue. Sed interdum aliquet augue, at pellentesque quam rhoncus vitae.

\nolinenumbers

\bibliography{plos}
% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here. See http://journals.plos.org/plosone/s/latex for 
% step-by-step instructions.
% 
% \begin{thebibliography}{10}

% \bibitem{bib1}
% Conant GC, Wolfe KH.
% \newblock {{T}urning a hobby into a job: how duplicated genes find new
%   functions}.
% \newblock Nat Rev Genet. 2008 Dec;9(12):938--950.

% \bibitem{bib2}
% Ohno S.
% \newblock Evolution by gene duplication.
% \newblock London: George Alien \& Unwin Ltd. Berlin, Heidelberg and New York:
%   Springer-Verlag.; 1970.

% \bibitem{bib3}
% Magwire MM, Bayer F, Webster CL, Cao C, Jiggins FM.
% \newblock {{S}uccessive increases in the resistance of {D}rosophila to viral
%   infection through a transposon insertion followed by a {D}uplication}.
% \newblock PLoS Genet. 2011 Oct;7(10):e1002337.

% \end{thebibliography}



\end{document}

